app/assets/javascripts/webchat/library.js

```diff
@@ -48,8 +52,33 @@
     }
 
     function apiSuccess (result) {
-      var validState  = API_STATES.indexOf(result.response) != -1
-      var state       = validState ? result.response : "ERROR"
+      //var validState  = API_STATES.indexOf(result.response) != -1
+
+
+      if(result.hasOwnProperty('inHOP')){
+        var validState  = API_STATES.indexOf(result.status.toUpperCase()) != -1
+        var state       = validState ? result.status : "ERROR"
+        if (result.inHOP == "true"){
+              if(result.availability == "true"){
```

Comment:
Non-blocker stylistic: we don't seem to have a linter setup in government-frontend, but can we consistently use two spaces for indentation?

Reply:
adjsuted tabbing